code/modules/materials/_materials.dm

```diff
@@ -285,6 +285,9 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 	var/bakes_into_material
 	var/bakes_into_at_temperature
 
+	// Used by leather.
```

Comment:
probably better to write a functional description of what it does/can be used for rather than just saying leather uses it, tbh